app/server/models/disbursement.py

```diff
@@ -46,7 +46,6 @@ class Disbursement(ModelBase, OneOrgBase):
     transfer_accounts = db.relationship(
         "TransferAccount",
         secondary=disbursement_transfer_account_association_table,
-        back_populates="disbursements",
```

Comment:
Deleting this one line made creating a new bulk disbursement (before approving-- the synchronous part) an order of magnitude faster.